### ISSUE TITLE
Release: Security Review

### ISSUE THREAD

**Issue by jlaura** [USGS-Astrogeology/FoundationalDataProducts#3]
This issue is for tracking the security review on the website.

The site is being built using Hugo, which renders to static HTML. That HTML is then hosted on AWS S3. Therefore, securing the built site is a matter of having the S3 bucket secured (one through IAM keys/policies with CHS) and through the security provided within the users browser. This latter concern is out of scope for the security review of this software.

[Hugo describes their security model](https://gohugo.io/about/security-model/) for the runtime that renders the site. As this runtime is not a part of the software being released and said runtime is executed by users in their own local environments (which in the case of USGS personnel are managed by USGS IT security folks), this security concern is out of scope for the library.

Basically, all the code here renders static HTML with no access to services or APIs. No data collection is made available. The runtime is used to build the site on a users computer and security of that system is out of scope.

**Comment by acpaquette** [comment]
After reviewing the Hugo security model linked in the issue, it is a secure method for site creation. Thus the html build in this repo is secure.
